examples/dsb_sn/wiring/main.go

```diff
@@ -7,14 +7,16 @@
 package main
 
 import (
+	_ "github.com/blueprint-uservices/blueprint/examples/dsb_sn/tests"
 	"github.com/blueprint-uservices/blueprint/examples/dsb_sn/wiring/specs"
+	_ "github.com/blueprint-uservices/blueprint/examples/dsb_sn/workflow/socialnetwork"
 	"github.com/blueprint-uservices/blueprint/plugins/cmdbuilder"
 	"github.com/blueprint-uservices/blueprint/plugins/workflow/workflowspec"
 )
 
 func main() {
-	// Configure the location of our workflow spec
-	workflowspec.AddModule("github.com/blueprint-uservices/blueprint/examples/dsb_sn/workflow")
+	// Configure the location of our workflow spec and tests
+	workflowspec.AddModule("github.com/blueprint-uservices/blueprint/examples/dsb_sn/workflow/socialnetwork")
```

Comment:
socialnetwork is the name of the package. The module and the corresponding go.mod file is in the workflow folder.